packages/web-app-files/src/search/sdk/list.ts

```diff
@@ -24,7 +24,12 @@ export default class List implements SearchList {
     )
 
     return plainResources.map((plainResource) => {
-      const resource = buildResource(plainResource)
+      let resourceName = decodeURIComponent(plainResource.name)
+      if (resourceName.startsWith('/dav')) {
+        resourceName = resourceName.slice(4)
+      }
+
+      const resource = buildResource({ ...plainResource, ...{ name: resourceName } })
```

Comment:
Could be simplified to this, no?

```suggestion
      const resource = buildResource({ ...plainResource, name: resourceName })
```

Reply:
sure ^^